elements/pfe-button/src/pfe-button.scss

```diff
@@ -0,0 +1,138 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: button;
+
+:host {
+  // Example local variable, using BEM format
+  // --pfe-button--Property: value;
+  display: inline-block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-variant="primary"]) button,
+button {
+  background-color: #06c;
+  color: #fff;
+  font-size: 16px;
+  padding: 8px 16px;
+  cursor: pointer;
+  border-radius: 3px;
+  border: 0;
+  position: relative;
+
+  &:after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    content: "";
+  }
+
+  &:hover,
+  &:focus {
+    background-color: #004080;
+  }
+}
+
+:host([pfe-variant="secondary"]) {
+  button {
+    background-color: transparent;
+    color: #06c;
+
+    &:after {
+      border: 1px solid #06c;
+      border-radius: 3px;
+    }
+
+    &:hover,
+    &:focus {
+      &:after {
+        border: 2px solid #06c;
```

Comment:
Changing border width on hover can cause layout issues, recommend box-shadow or outline instead. They can do the same visual effect without layout reflow

Reply:
If you do this, I'd make replace all borders with whatever you choose, probably means you'll have to add to padding.

Reply:
@wesruv Using an :after element positioned absolutely doesn't cause any layout issues since it's just sitting on top.

Reply:
ahh, gotcha